src/AppInstallerRepositoryCore/Microsoft/PinningIndex.cpp

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+#include "PinningIndex.h"
+#include "SQLiteStorageBase.h"
+#include "Schema/Pinning_1_0/PinningIndexInterface.h"
+
+namespace AppInstaller::Repository::Microsoft
+{
+    PinningIndex PinningIndex::CreateNew(const std::string& filePath, Schema::Version version)
+    {
+        AICLI_LOG(Repo, Info, << "Creating new Pinning Index [" << version << "] at '" << filePath << "'");
+        PinningIndex result{ filePath, version };
+
+        SQLite::Savepoint savepoint = SQLite::Savepoint::Create(result.m_dbconn, "pinningIndex_createNew");

```

Comment:
```suggestion
        SQLite::Savepoint savepoint = SQLite::Savepoint::Create(result.m_dbconn, "pinningindex_createnew");

```

nit: update all strings used by sqlite to lower case